edwin/client/static/js/components/Timeline.js

```diff
@@ -82,6 +99,22 @@ Timeline.propTypes = {
   params: React.PropTypes.object.isRequired,
 };
 
+class Indicator extends React.Component {
+  render() {
+    let state = this.props.state.toJS().state;
+    if (state !== LoadingStates.DONE.toString()) {
+      return (
+        <span className="Indicator">{state} ...</span>
+      );
+    } else {
+      return (<span></span>);
```

Comment:
It would probably be better to return `null` here, which will tell React to simply put nothing here.

Also, if you want to keep this empty span, the parens aren't needed (so you could do `return <span></span>;`), and tags with no children can be written in a short form, like `return </span>;`. The parens are only needed when the JSX tag will extend multiple lines.